### PR TITLE
Allowing stack creation from API

### PR DESCRIPTION
**Pull request by lxfontes**
Seems like permission `write:stack` wasn't being used anywhere, so I used it instead of creating `create:stack`.

### CODE REVIEW

**Comment by lxfontes** [comment]
cc @byroot @wfarr

**Comment by camilo** [comment]
👍

File: app/controllers/shipit/api/stacks_controller.rb [hunk 1]
```diff
@@ -16,6 +23,10 @@ def show
       def stack
         @stack ||= stacks.from_param!(params[:id])
       end
+
+      def create_params
```
Comment (byroot) [inline app/controllers/shipit/api/stacks_controller.rb]: For the API shipit uses explicit-parameters. You can look at other API controllers or webhook controller.

Also for creation parame shouldn't be nested under :stack

**Comment by casperisfine** [comment]
Thx!

**Pull request merged**